app-emulation/nemu/nemu-3.3.0.ebuild

```
@@ -0,0 +1,80 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake linux-info
+
+MY_PV="${PV/_rc/-RC}"
+
+DESCRIPTION="ncurses interface for QEMU"
```

Captial first letter:
```suggestion
DESCRIPTION="Ncurses interface for QEMU"
```

---

Fixed